<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>610</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>907</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>907</width>
    <height>610</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-40</y>
      <width>911</width>
      <height>651</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>280</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Placeholder 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>340</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Placeholder 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>400</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Placeholder 3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>460</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Placeholder 4</string>
    </property>
   </widget>
   <widget class="QLabel" name="titulo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>881</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CAPACITY MEASUREMENT GRAPHIC USER INTERFACE</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../Desktop/gui test/source.qrc"/>
 </resources>
 <connections/>
</ui>
